docs/config.md

```diff
@@ -13,57 +15,100 @@ body of the hosting page:
 <script async src="https://hypothes.is/embed.js"></script>
 ```
 
+Not all configuration settings can be set in this way, some must be
+[set using JavaScript](#configuring-the-client-using-javascript) (see below).
+
 **N.B.** The body of the `.js-hypothesis-config` tag must be [valid
 JSON](http://jsonlint.com/) -- invalid JSON will cause the entire config object
 to be ignored.
 
-Config keys
------------
+### Configuring the client using JavaScript
+
+Alternatively, the Hypothesis client can be configured from the hosting page by
+providing a JavaScript function named `window.hypothesisConfig` that returns
+a configuration object. Some configuration settings can _only_ be set from
```

Comment:
This could be more specific, eg. "Settings which specify callbacks or event handlers can only be set from JavaScript".